editor/gui/editor_scene_tabs.cpp

```diff
@@ -135,6 +135,15 @@ void EditorSceneTabs::_scene_tab_input(const Ref<InputEvent> &p_input) {
 	}
 }
 
+void EditorSceneTabs::unhandled_input(const Ref<InputEvent> &p_event) {
+	ERR_FAIL_COND(p_event.is_null());
```

Comment:
```suggestion
```
This check is unnecessary. You are already checking for event validity below.

Reply:
Fixed